CPTextView/Frameworks/TextView/CPTypesetter.j

```diff
@@ -352,26 +381,25 @@ var CPSystemTypesetterFactory;
                 else
                     rangeWidth += 28;   //FIXME
             }  // fallthrough intentional
-            case  ' ':
+            case 32: // ' '
                 wrapRange = CPMakeRangeCopy(lineRange);
                 wrapRange._height = _lineHeight;
                 wrapRange._base = _lineBase;
                 wrapWidth = rangeWidth;
                 break;
+            case 10:
```

Comment:
Yeah, it is an optimisation for time critical code. Working with char codes is faster then working with strings. This is copied from the Object-J parser. Almost all parsers work like this to gain speed. I do agree that it is nicer to encapsulate newline semantics like you did but this is all for the speed.